src/main/scala/io/chrisdavenport/fuuid/FUUID.scala

```diff
@@ -8,6 +8,7 @@ import java.util.UUID
 final class FUUID private (private val uuid: UUID){
   def equals(that: FUUID): Boolean = 
     this.uuid equals that.uuid
+  override def hashCode: Int = uuid.hashCode
```

Comment:
Great, but `equals` also should be overridden:
```
  override def equals(obj: scala.Any): Boolean = obj match {
    case that: FUUID => this.uuid equals that.uuid
    case _ => false
  }
```
And please add some tests for sure:
```
    "Has proper hashCode and equals" in {
      val uuid = java.util.UUID.fromString("00000000-075b-cd15-0000-0000075bcd15")
      val fuuid: AnyRef = FUUID.fromString("00000000-075b-cd15-0000-0000075bcd15").right.get
      FUUID.fromUUID(uuid).equals(fuuid) must_=== true
      FUUID.fromUUID(uuid).hashCode() must_=== uuid.hashCode()
    }
```
```